src/components/TopMenu.tsx

```diff
@@ -154,8 +158,20 @@ function RewardsButton({
 }: {
   setCurrentModal: React.Dispatch<React.SetStateAction<string | null>>
 }): ReactElement | null {
+  const { chainId } = useActiveWeb3React()
+  const gaugesAreActive = areGaugesActive(chainId)
   const rewardBalances = useContext(RewardsBalancesContext)
-  const formattedTotal = formatBNToShortString(rewardBalances.total, 18)
+  const userState = useContext(UserStateContext)
+  const totalSdlFromGauges = Object.values(
+    userState?.gaugeRewards ?? {},
+  ).reduce((sum, { claimableSDL }) => sum.add(claimableSDL), Zero)
+  const formattedTotal = formatBNToShortString(
+    gaugesAreActive
+      ? rewardBalances.total.add(totalSdlFromGauges) // adding against retroactive amount
```

Comment:
It might be easier just to put this same logic where `rewardBalances.total` is stored in the first place - that way you just update it once and its represented everywhere